pandas/tests/formats/test_format.py

```diff
@@ -3938,6 +3938,15 @@ def test_period(self):
         self.assertEqual(str(df), exp)
 
 
```

Comment:
yep, np. Though I don't think its a big deal to completely drop nose (but sure can do later).